README.md

```diff
@@ -40,349 +40,120 @@ A Container Storage Interface ([CSI](https://github.com/container-storage-interf
 
 **Requirements:**
 
-* k8s 1.12
-  * `--allow-privileged` flag must be set to true for both the API server and the kubelet
-  * `--feature-gates=CSINodeInfo=true,CSIDriverRegistry=true,VolumeSnapshotDataSource=true` feature gate flags must be set to true for both the API server and the kubelet
-* k8s 1.13
-  * `--allow-privileged` flag must be set to true for the API server
-  * `--feature-gates=VolumeSnapshotDataSource=true` feature gate flags must be set to true for the API server
-* k8s 1.14
-  * `--allow-privileged` flag must be set to true for the API server
-  * `--feature-gates=ExpandCSIVolumes=true,ExpandInUsePersistentVolumes=true` feature gate flags must be set to true for both the API server and kubelet.
+Kubernetes 1.12
+ * `--allow-privileged` flag must be set to true for both the API server and the kubelet
+ * `--feature-gates=CSINodeInfo=true,CSIDriverRegistry=true,VolumeSnapshotDataSource=true` feature gate flags must be set to true for both the API server and the kubelet
+ 
+Kubernetes 1.13
+ * `--allow-privileged` flag must be set to true for the API server
+ * `--feature-gates=VolumeSnapshotDataSource=true` feature gate flags must be set to true for the API server
+ 
+Kubernetes 1.14
+ * `--allow-privileged` flag must be set to true for the API server
+ * `--feature-gates=ExpandCSIVolumes=true,ExpandInUsePersistentVolumes=true` feature gate flags must be set to true for both the API server and kubelet
 
-#### 1. Create a secret with your array details:
+### Deploying the HPE CSI Driver for Kubernetes
+The following example walks through deployment of the driver with a HPE Nimble Storage CSP backend. Replace any Nimble references with your particular CSP nomenclature.
 
-Replace the password string (`YWRtaW4=`) with a base64 encoded version of your password and replace the arrayIp with your array IP address and save it as `secret.yaml`:
+#### Create a secret with your platform details
+Replace the password string (`YWRtaW4=`) with a base64 encoded version of your password and replace the `backend` with the IP address of the CSP backend and save it as `secret.yaml`:
 
 ```
 apiVersion: v1
 kind: Secret
 metadata:
   name: nimble-secret
-  namespace: default
+  namespace: kube-system
```

Comment:
This doesn't need to be in kube-system  namespace

Reply:
The `default` namespace is non-mandatory for downstream distros. All our bits runs in `kube-system`. Unless we create our own namespace, this is the prettiest option.